### ISSUE TITLE
Tests fail on current master

### ISSUE THREAD

**Issue by arokem** [arokem/python-matlab-bridge#184]
```
D-173-250-157-207:~  $nosetests pymatbridge
...........................EE.......
======================================================================
ERROR: test_publish.test_convert_mfile
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/arokem/anaconda3/lib/python3.4/site-packages/nose-1.3.4-py3.4.egg/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/tests/test_publish.py", line 44, in test_convert_mfile
    publish.convert_mfile(MFILE)
  File "/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/publish.py", line 125, in convert_mfile
    lines = mfile_to_lines(mfile)
  File "/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/publish.py", line 53, in mfile_to_lines
    with open(mfile) as fid:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/tests/test_publish.m'

======================================================================
ERROR: test_publish.test_mfile_to_lines
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/arokem/anaconda3/lib/python3.4/site-packages/nose-1.3.4-py3.4.egg/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/tests/test_publish.py", line 54, in test_mfile_to_lines
    lines = publish.mfile_to_lines(MFILE)
  File "/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/publish.py", line 53, in mfile_to_lines
    with open(mfile) as fid:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/tests/test_publish.m'

----------------------------------------------------------------------
Ran 36 tests in 205.857s

FAILED (errors=2)
D-173-250-157-207:~  $cd ~/source/python-matlab-bridge/

```

**Comment by arokem** [comment]
Seems to me like an installation issue. Looking into it!

**Comment by jjangsangy** [comment]
It looks the file not found error isn't able to find the `.m` from your `site-packages` directory.

```
FileNotFoundError: [Errno 2] No such file or directory: '/Users/arokem/anaconda3/lib/python3.4/site-packages/pymatbridge/tests/test_publish.m'
```

I went and checked, and it doesn't look like we include the `*.m` files in the `package_data` from these lines.

https://github.com/arokem/python-matlab-bridge/blob/master/pymatbridge/version.py#L78-L85

**Comment by arokem** [comment]
Yeah - there's already a PR for that:
https://github.com/arokem/python-matlab-bridge/pull/185

**Comment by jjangsangy** [comment]
Cool, didn't see that one.

Actually the reason I brought this up is because the `test_publish.py` and `test_publish.m` files are also giving me some build issues as well, but likely for another reason.

Was wondering how the `test_convert_mfile`, `test_lines_to_notebook`, and `test_mfile_to_lines` worked?

**Comment by arokem** [comment]
On Sun, May 3, 2015 at 11:41 PM, Sang Han <notifications@github.com> wrote:

> Cool, didn't see that one.
>
> Actually the reason I brought this up is because the test_publish.py and
> test_publish.m files are also giving me some build issues as well, but
> likely for another reason.
>
> Was wondering how the test_convert_mfile, test_lines_to_notebook, and
> test_mfile_to_lines worked?
>

The general idea is to process the text in an m file (a script), so that it
renders as a notebook that can be opened in a Jupyter notebook with the
matlab kernel. Does that answer your question?

**Comment by jjangsangy** [comment]
Yes! Thanks, that's perfect high level overview!

**Comment by arokem** [comment]
Closed through #185
